pypiserver/config.py

```diff
@@ -382,6 +396,15 @@ def get_parser() -> argparse.ArgumentParser:
             "index."
         ),
     )
+    run_parser.add_argument(
+        "--health-endpoint",
+        default=DEFAULTS.HEALTH_ENDPOINT,
+        type=health_endpoint_arg,
+        help=(
+            "Configure a custom liveness endpoint. It always returns 200 Ok if "
+            "the service is up. Otherwise it means that the service is dead."
```

Comment:
```suggestion
            "the service is up. Otherwise, it means that the service is not responsive."
```

Just as an alignment :)